ambari-web/app/controllers/main/admin/kerberos/step2_controller.js

```diff
@@ -164,8 +169,8 @@ App.KerberosWizardStep2Controller = App.WizardStep7Controller.extend(App.KDCCred
 
     typeSettings.configNames.forEach(function (_configName) {
       var config = configs.findProperty('name', _configName);
-      if (config) {
-        config.isVisible = kdcType === typeSettings.type;
+      if (config && (kdcType === typeSettings.type)) {
+        config.isVisible = true;
```

Comment:
If this line is not changed, new bug will be introduced because `ldap_url`, `container_dn` is common for both 'ad' and 'ds' in `kerberosConfigMap`.
Input box for `ldap_url` and `container_dn` will not be shown.